### PR TITLE
Fixes #469 - query results error out if no custom font is set

### LINKED ISSUE #469: Running a query results in the error "Cannot match against 'undefined' or 'null'."
When checking out the project fresh on Master using Node 8 and trying to run a select query on either Postgres or MySQL causes this error in the console:

`"Cannot match against 'undefined' or 'null'."`

Screen Shot 2019-05-02 at 8 36 31 PM

### PR DESCRIPTION
**Pull request by rathboma**
- if no custom font has been set from the settings modal you are unable to see query results
- this fixes the issue by defaulting the custom font to the default font

Fixes #469

### CODE REVIEW

**Comment by rathboma** [comment]
Not sure what this build failure is about -- seems maybe related to a library upgrade?

**Comment by rathboma** [comment]
Ah, looks like the Travis build is running on node 6 - `- nvm install 6` in the config, but there are some async functions in the code.

File: src/renderer/components/query-result-table.jsx [hunk 1]
```diff
@@ -336,13 +336,11 @@ export default class QueryResultTable extends Component {
    * It gives a better UX since the column adapts better to the table width.
    */
   resolveCellWidth(fieldName, fields, rows) {
-    const { customFont } = this.props.config.data.customFont;
-    const font = `14px '${customFont}', 'Helvetica Neue', Arial, Helvetica, sans-serif`;
+    const customFont = this.props.config.data.customFont || 'Helvetica Neue';
+    const font = `14px '${customFont}' 'Helvetica Neue', Arial, Helvetica, sans-serif`;
```
Comment (maxcnunes) [inline src/renderer/components/query-result-table.jsx]: You removed the `,` from the customFont. I think the comma is required there.

Comment (rathboma) [reply src/renderer/components/query-result-table.jsx]: yes you are correct. I will fix that.

**Comment by MasterOdin** [comment]
Manually merged in df7005281cfcd9a50eb625b9ac0010175d650cec

**Pull request closed without merging**